src/test/scala/org/camunda/feel/impl/builtin/BuiltinConversionFunctionsTest.scala

```diff
@@ -211,7 +211,21 @@ class BuiltinConversionFunctionsTest
     eval(""" string(date and time("2012-12-25T11:00:00+02:00")) """) should be(
       ValString("2012-12-25T11:00:00+02:00"))
   }
+  it should "convert zero-length days-time-duration" in {
+    eval(""" string(@"-PT0S") """) should be(ValString("PT0S"))
+    eval(""" string(@"P0D") """) should be(ValString("PT0S"))
+    eval(""" string(@"PT0H") """) should be(ValString("PT0S"))
+    eval(""" string(@"PT0H0M") """) should be(ValString("PT0S"))
+    eval(""" string(@"PT0H0M0S") """) should be(ValString("PT0S"))
+    eval(""" string(@"P0DT0H0M0S") """) should be(ValString("PT0S"))
+  }
```

Comment:
:x: Actually, the expected value of a zero-length duration is `P0D`. :sweat_smile: 

I added a comment [on the issue](https://github.com/camunda/feel-scala/issues/598#issuecomment-1527173279).

Reply:
🙈 Yes, I think I can handle that. 😅